src/components/calcite-action-bar/calcite-action-bar.scss

```diff
@@ -1,6 +1,6 @@
 :host {
   height: 100%;
-  display: flex;
+  display: inline-flex;
```

Comment:
The action bar was fillings its container. It seemed that behavior was *undesired* so I changed this to inline-flex to remedy that.